kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiRestFrontendService.scala

```diff
@@ -116,6 +116,13 @@ class KyuubiRestFrontendService(override val serverable: Serverable)
   }
 
   private def installWebUI(): Unit = {
+    if (!conf.get(FRONTEND_REST_ENABLE_WEBUI)) {
```

Comment:
prefer to judge on the caller's side to make this function clear. BTW, what will happen if we don't inject the `enable.html` page?

Reply:
If we don't inject the `enable.html`, we just get a 404 page.